rust/Cargo.toml

```
@@ -1,15 +1,15 @@
 [package]
 name = "nitor-vault"
-version = "0.5.0"
+version = "0.6.0"
```

```suggestion
version = "0.5.1"
```


Any behavior doesn't seem to change, as under the hood `aws-config` uses `DefaultRegionChain` which in turn first tries `EnvironmentVariableRegionProvider`, which tries to load `AWS_REGION` from env.